### ISSUE TITLE
PyTorch OP Doc List

### ISSUE THREAD

**Issue by PaParaZz1** [opendilab/DI-treetensor#2]
# P0
* [x] cpu
* [x] cuda
* [x] to
* [x] torch.zeros_like
* [x] torch.randn_like
* [x] torch.ones_like
* [x] torch.zeros
* [x] torch.randn
* [x] torch.randint
* [x] torch.ones
* [x] cat
* [x] reshape
* [x] split
* [x] squeeze
* [x] stack
* [x] unsqueeze
* [x] where
* [x] abs
* [x] add
* [x] clamp
* [x] div
* [x] exp
* [x] log
* [x] sqrt
* [x] sub
* [x] sigmoid
* [x] pow
* [x] mul
* [ ] argmax
* [ ] argmin
* [x] all
* [x] any
* [x] max
* [x] min
* [x] dist
* [x] mean
* [x] std
* [x] sum
* [x] eq
* [x] ge
* [x] gt
* [x] le
* [x] lt
* [x] ne
* [x] clone
* [x] dot
* [x] matmul
* [x] mm
# P1
* [x] numel
* [x] torch.randint_like
* [x] torch.full_like
* [x] torch.empty_like
* [x] torch.full
* [x] torch.empty
* [x] chunk
* [ ] gather
* [x] index_select
* [x] masked_select
* [ ] scatter
* [ ] tile
* [ ] unbind
* [x] ceil
* [x] exp2
* [x] floor
* [x] log10
* [x] log2
* [x] neg
* [x] round
* [x] sign
* [ ] bmm
# P2
* [ ] logsumexp
* [ ] median
* [x] norm
* [ ] prod
* [ ] unique
* [ ] argsort
* [x] isfinite
* [x] isinf
* [x] isnan
* [ ] sort
* [ ] topk
* [ ] cdist
* [ ] flip
* [ ] addbmm
* [ ] addmm

**Comment by HansBug** [comment]
Ahhh, got it.... still so many 😭 

I though that I need a script to auto generate the running result in the doc example. Run and copy it one by one is terrible

**Comment by HansBug** [comment]
All the ops has been supported by dynamic features.
